### PR TITLE
data: Andrewyithub

### PR DESCRIPTION
**Pull request by Andrewyithub**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Andrewyithub.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Andrew Y",
+  "bio": "Full Stack and Open Source. Travel & Fitness enthusiast. ✌️"
```
Comment (rizahmeds) [inline data/Andrewyithub.json:3]: You missed a comma at the end of line number 3.

Comment (SaraJaoude) [inline data/Andrewyithub.json:3]: adding missing comma:
```suggestion
  "bio": "Full Stack and Open Source. Travel & Fitness enthusiast. ✌️",
```

File: data/Andrewyithub.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    {
+      "name": "Let's connect on Linkedin",
+      "url": "https://www.linkedin.com/in/andrewydev/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Check out my portfolio",
+      "url": "https://andrewy.me",
+      "icon": "FaNewspaper"
+    },
```
Comment (SaraJaoude) [inline data/Andrewyithub.json:24]: removing extra comma:
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**